cram_pr2/cram_pr2_environment_manipulation/src/action-designators.lisp

```diff
@@ -29,6 +29,17 @@
 
 (in-package :pr2-em)
 
+(defun get-container-pose-and-transform (name)
+  (let* ((name-rosified (roslisp-utilities:rosify-underscores-lisp-name name))
+         (urdf-pose (get-urdf-link-pose name-rosified))
+         (pose (cl-tf:transform-pose-stamped cram-tf:*transformer*
```

Comment:
Please use ``cram-tf:ensure-pose-in-frame`` instead of ``cl-tf:transform-pose-stamped``. And maybe use zero time.